app/src/features/apiClient/slices/workspaceView/helpers/ApiClientContextService/ApiClientContextService.ts

```
@@ -0,0 +1,276 @@
+    const repo = this.createRepository({ workspaceId, workspaceMeta: workspace.meta, user: userDetails });
+    await repo.validateConnection();
+
+    const store = this.createStore(workspaceId);
+
+    const result = await this.extractSetupDataFromRepository(repo);
+    await this.hydrateInPlace({
+      workspaceId: workspace.id || 'null',
+      entitiesToHydrate: { apiClientRecords: result.apiClientRecords.records },
+      store,
+    });
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)